pkg/cmd/gtctl/cluster/get/test_data/pids/a/pid

```diff
@@ -0,0 +1 @@
+123
```

Comment:
nitpick: it's better to add the newline.

Reply:
sure